Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -639,35 +618,43 @@ extension BrowserViewController: WKNavigationDelegate {
     }
```

Comment:
Hm, do we even want this check here? Observers might be interested in `LocationChange` events even if that event happens on a non-selected tab. In fact, the only `LocationChange` observer we have right now is the one that adds the history entry to the DB, which we wouldn't want to exclude background tabs for, right?